R/utils_pca_efa.R

```diff
@@ -1,62 +1,73 @@
-#' @export
-sort.parameters_efa <- function(x, ...) {
-  .sort_loadings(x)
+# summary -----------------------------------------------------------------
+
+#' @keywords internal
+.transpose_summary_efa <- function(object, ...){
+
+
 }
 
-#' @export
-sort.parameters_pca <- sort.parameters_efa
 
 #' @export
 summary.parameters_efa <- function(object, ...) {
-  insight::print_color("# (Explained) Variance of Principal Components\n\n", "blue")
 
   x <- attributes(object)$summary
-  x$Variance_Explained <- x$Variance / sum(x$Variance)
-  x$Std_Dev <- attributes(object)$model$sdev
-
-  col_names <- x$Component
+  # x$Std_Dev <- attributes(object)$model$sdev
```

Comment:
@strengejacke what is this for? Is it actually used somewhere? Couldn't find any instances

Reply:
Not sure, that must be your code... 😕

Reply:
don't think so, would have never call it "Std_Dev" :grin: